### PR TITLE
Add a workaround check for sshd about curve25519 error

### PR DESCRIPTION
**Pull request by jouyingbin**
Description:
Currently the curve25519 is not FIPS approved, it's not
available seems intended, this fix is for ENV mode error,
Kernel mode does not show the error.

- Related ticket: https://progress.opensuse.org/issues/104415
- Needles: NA
- Verification run: 
  (SP4 ENV mode) https://openqa.suse.de/tests/7916259#step/sshd/122
  (SP4 Kernel mode) https://openqa.suse.de/tests/7916260
  (SP3 Maintenance) https://openqa.suse.de/t7918705
  (openSUSE TW) https://openqa.opensuse.org/t2110300

### CODE REVIEW

**Review by lilyeyes — APPROVED** [review]
lgtm.

**Review by rfan1 — APPROVED** [review]

**Review by ShawnHa0 — APPROVED** [review]
LGTM

File: tests/console/sshd.pm [hunk 1]
```diff
@@ -121,7 +121,13 @@ sub run {
     assert_script_run 'until ss -tulpn|grep sshd|egrep "4242|5252";do sleep 1;done';
 
     # Scan public keys on forwarded ports
-    assert_script_run "ssh-keyscan -p 4242 localhost >> ~/.ssh/known_hosts";
+    # Add a workaround about bsc#1193275 in FIPS test
+    my $output = script_output("ssh-keyscan -p 4242 localhost >> ~/.ssh/known_hosts", proceed_on_failure => 1);
+    if ($output =~ /choose_kex: unsupported KEX method curve25519-sha256/) {
+        record_soft_failure('bsc#1193275', "Currently the curve25519 is not FIPS approved");
+        return;
```
Comment (Amrysliu) [inline tests/console/sshd.pm:128]: what does the `return` use for? exit the test?

Comment (jouyingbin) [reply tests/console/sshd.pm:128]: hmm, I think it is not necessary, right?

Comment (Amrysliu) [reply tests/console/sshd.pm:128]: If you want to the following tests continue run, it is not necessary.

Comment (jouyingbin) [reply tests/console/sshd.pm:128]: good comment, I think I should remove it, thanks.

Comment (jouyingbin) [reply tests/console/sshd.pm]: DONE.

**Review by Amrysliu — APPROVED** [review]

**Pull request merged**